flocker/cli/script.py

```diff
@@ -250,9 +250,49 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+
+    """
+    longdesc = """flocker does nothing (yet).
+
+    """
+
+    synopsis = ("Usage: flocker [OPTIONS] "
+                "\n"
+                "If you have any issues or feedback, you can talk to us: "
+                "http://docs.clusterhq.com/en/latest/gettinginvolved/"
+                "contributing.html#talk-to-us")
+
+
+@implementer(ICommandLineScript)
+class CLIScript(object):
+    """
+    A command-line script to interact with a cluster via the API.
+    """
+    def main(self, reactor, options):
+        """
+        See :py:meth:`ICommandLineScript.main` for parameter documentation.
+
+        :return: A ``Deferred`` which fires when the deployment is complete or
+                 has encountered an error.
+        """
+        return succeed(None)
+
+
 def flocker_deploy_main():
     return FlockerScriptRunner(
         script=DeployScript(),
         options=DeployOptions(),
         logging=False,
     ).main()
+
+
+def flocker_cli_main():
+    return FlockerScriptRunner(
+        script=CLIScript(),
+        options=CLIOptions(),
+        logging=False,
```

Comment:
My understanding is that we are taking the approach now of the more logging the better. Please either add basic logging or link to a follow-up issue to do so.